solidity/dashboard/jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "typeAcquisition": {
+        "include": ["jest"]
+    }
+    
+}
```

Comment:
New line is missing there. You can also check if there are any other js lint errors just in case you didn't 😉